languages/lang-ja/Naming.ja.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<naming Lang="ja">
+<naming Lang="ja" Load="Merge">
 
   <namestyles>
 
-    <namestyle Name="Qudish (ja)" Format="AsIs">
-      <scopes>
+    <namestyle Name="Qudish" Format="AsIs">
+      <!-- <scopes>
```

Comment:
we can remove/comment the scopes entirely, unless we want to change something about them